release-notes/7.0/supported-os.json

```diff
@@ -0,0 +1,64 @@
+{
+  "channel-version": "7.0",
+  "last-updated": "2024-02-28",
+  "glibc": [
+    {
+      "name": "CentOS",
+      "supported-cycles": [
+        {
+          "cycle": "7",
+          "architectures": ["x64"],
+          "required-version": "2.17"
```

Comment:
I prefer `min-version` over `required-version` since that more accurately articulates the contract. We could equally just say `version` and document somewhere that this is the version we target. That's probably the nicest.

Reply:
We also have a C++ standard library dependency. We should add that, too.

Reply:
I like `min-version` as it's more expressive then `version`. Do you have an example for libc? I think I didn't see it in the `supported-os.md`.

Reply:
Here: https://github.com/dotnet/core/pull/9210#discussion_r1560391672

In that case, writing `min-version` and `source` seems odd.